_config.yml

```diff
@@ -13,8 +13,8 @@ exclude_from_localizations: ['javascript', 'images', 'css']
 
 # Contact form production values (overridden in Cloud.gov Pages for preview branches)
 contact_form_captcha_enabled: true
-contact_form_action: 'https://webto.salesforce.com/servlet/servlet.WebToCase?encoding=UTF-8'
-contact_form_orgid: '00DU0000000Leux'
+contact_form_action: 'https://gsa-peo--peopc1stgp.sandbox.my.salesforce.com/servlet/servlet.WebToCase?encoding=UTF-8'
+contact_form_orgid: '00DHv0000008oc1'
```

Comment:
Do we need to make these configuration changes in the Cloud.gov Pages site configuration?

Reply:
We do not. I read the instructions incorrectly and did not realize that config change for testing needed to be made in Cloud.gov, not in the file itself. Once all the checks are completed from Salesforce, I am going to revert this and the recaptcha change back before merging

Reply:
Whoops, I should have been more specific: Specifically, I'm wondering if the configuration for the _preview sites_ is outdated, in case the Salesforce team has permanently updated the values. I was under the impression that our preview sites were already configured to point to their sandbox.

Reply:
Ah, gotcha.

The configuration `contact_form_orgid` changes when there is a new test, so it will be current until the next SF test. The contact form `action` stays the same and is always pointed to their sandbox.

Reply:
Ok, makes sense 👍 I also see that the OrgID has been updated in both the process doc and the site configuration, so all looks good to me 👍